tests/test_iam.py

```diff
@@ -81,19 +81,25 @@ class TestIAM(unittest.TestCase):
                 {
                     "server_certificate":
                     {
-                        "certificate_body": ("-----BEGIN CERTIFICATE-----"
-                                             "CERT1CERT1CERT1CERT1CERT1CER"
-                                             "-----END CERTIFICATE-----"),
-                        "certificate_chain": ("-----BEGIN CERTIFICATE-----"
-                                              "CHAIN1CHAIN1CHAIN1CHAIN1CHA"
-                                              "-----END CERTIFICATE-----"),
-                        "certificate_key": ("-----BEGIN PRIVATE KEY-----"
-                                            "KEY1KEY1KEY1KEY1KEY1KEY1KEY"
-                                            "-----END PRIVATE KEY-----"),
-                        }
+                        "certificate_body": (
+                            "-----BEGIN CERTIFICATE-----"
+                            "CERT1CERT1CERT1CERT1CERT1CER"
+                            "-----END CERTIFICATE-----"
+                        ),
+                        "certificate_chain": (
+                            "-----BEGIN CERTIFICATE-----"
+                            "CHAIN1CHAIN1CHAIN1CHAIN1CHA"
+                            "-----END CERTIFICATE-----"
+                        ),
+                        "certificate_key": (
+                            "-----BEGIN PRIVATE KEY-----"
+                            "KEY1KEY1KEY1KEY1KEY1KEY1KEY"
+                            "-----END PRIVATE KEY-----"
+                        ),
                     }
                 }
             }
+        }
```

Comment:
Can you remove this change from the PR - it's unrelated to the change we're making here and will just confuse git history